### ISSUE TITLE
Update 'Adding WebView to your Flutter app' codelab

### ISSUE THREAD

**Issue by WangHepler** [flutter/flutter#117808]
```
lib/main.dart:26:19: Error: Couldn't find constructor 'WebView'.
      body: const WebView(
                  ^^^^^^^
```

I get the above error after following the "https://codelabs.developers.google.com/codelabs/flutter-webview#2" step-by-step.
Who knows the reason?Please tell me,thank you so much!

**Comment by danagbemava-nc** [comment]
Hi @WangHepler, there was a breaking change to the `webview_flutter` plugin recently so the codelab is currently out of date. Kindly go to https://pub.dev/packages/webview_flutter and read the instructions to see how to use the plugin. 

The codelab will have to be updated because of the changes. 

/cc @domesticmouse

**Comment by domesticmouse** [comment]
Fix is in cl/496865324

**Comment by domesticmouse** [comment]
This should now be fixed.
